cypress.config.js

```diff
@@ -0,0 +1,12 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  e2e: {
+    env: {
+      baseUrl: "https://mherholdt94.github.io/social-media-client/",
```

Comment:
```suggestion
      baseUrl: "/",
```

The only code you should test is the specific code in this branch. By testing a remote URL you will always be testing a static deployment.